src/App.tsx

```diff
@@ -15,47 +22,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+
+function getPreperedGoods(goods: string[], sortMethod: SortTypes | string, sortDirection: SortTypes | string): string[] {
```

Comment:
<img width="674" alt="image" src="https://github.com/mate-academy/react_list-of-goods-hooks/assets/70220667/10c928b7-665e-474d-9b1d-0573a54025ca">